lib/Runtime/Library/JavascriptExternalFunction.cpp

```diff
@@ -424,9 +430,15 @@ namespace Js
             TTD::TTDNestingDepthAutoAdjuster logPopper(scriptContext->GetThreadContext());
             TTD::NSLogEvents::EventLogEntry* callEvent = elog->RecordExternalCallEvent(externalFunction, scriptContext->GetThreadContext()->TTDRootNestingCount, args.Info.Count, args.Values, true);
```

Comment:
Do these TTD events need to be updated with the `new.target` information? Right now only the arguments are passed in.

Reply:
I think we need to do something more here for TTD cases because `RecordExternalCallEvent` doesn't even record the flags for the call at all right now. `new.target` is located in the args, so not sure it should be handled specially but we should probably care if the function was called as a new expression or not.

Reply:
Since the TTD tests are passing again, this may not need to change. I think that since the call flags aren't directly visible in JS, when replaying we probably don't need to know the flags, the external function is a black-box that has some js-visible side effects and a return value, all of which are currently recorded.